docker/compose.yaml

```diff
@@ -0,0 +1,27 @@
+services:
+  sglang:
+    image: lmsysorg/sglang:latest
+    container_name: sglang
+    volumes:
+      - ${HOME}/.cache/huggingface:/root/.cache/huggingface
+    restart: always
+    network_mode: host
+    # Or you can only publish port 30000
+    # ports:
+    #   - 30000:30000
+    entrypoint: python3 -m sglang.launch_server
+    command:
+      --model-path meta-llama/Meta-Llama-3.1-8B-Instruct
```

Comment:
It looks like HF_TOKEN hasn't been set. How can we download the model without it?

Reply:
Sorry I use a local model for testing so I missed it.